src/shogun/base/init.h

```diff
@@ -109,17 +109,13 @@ void set_global_rand(CRandom* rand);
  */
 CRandom* get_global_rand();
 
-/** set the global linalg library object
- *
- * @param linalg linalg object to use
- */
-void set_global_linlg(SGLinalg* linalg);
-
+#ifndef SWIG // SWIG should skip this part
 /** get the global linalg library object
  *
  * @return linalg object
  */
 SGLinalg* get_global_linlg();
```

Comment:
I think this is the reason some travis tests kept failing..the typo was there since the very beginning..